cmd/kubeadm/app/cmd/phases/util.go

```diff
@@ -18,6 +18,7 @@ package phases
 
 import (
 	"k8s.io/component-base/version"
+	"os"
```

Comment:
```suggestion	
	"os"
	
	"k8s.io/component-base/version"
```

we try to keep stdlib / k8s / local imports separate.